examples/templates/ipoib-rdma-shared/10-nicclusterpolicy.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: mellanox.com/v1alpha1
+kind: NicClusterPolicy
+metadata:
+  name: nic-cluster-policy
+spec:
+  ofedDriver:
+    image: doca-driver
+    repository: |doca-driver-repository|
+    version: |doca-driver-version|
+  rdmaSharedDevicePlugin:
+    image: k8s-rdma-shared-dev-plugin
+    repository: |k8s-rdma-shared-dev-plugin-repository|
+    version: |k8s-rdma-shared-dev-plugin-version|
+    config: |
+      {
+        "configList": [
+          {
+            "resourceName": "rdma_shared_device_a",
+            "rdmaHcaMax": 63,
+            "selectors": {
+              "ifNames": ["ibs1f0"]
+            }
+          },
+          {
+            "resourceName": "rdma_shared_device_b",
+            "rdmaHcaMax": 63,
+            "selectors": {
+              "ifNames": ["ibs1f1"]
+            }
+          },
+          {
+            "resourceName": "rdma_shared_device_c",
+            "rdmaHcaMax": 63,
+            "selectors": {
+              "ifNames": ["ibs2f0"]
+        ]
+      }
+  nvIpam:
+    image: nvidia-k8s-ipam
+    repository: |nvidia-ipam-repository|
```

Comment:
just a suggestions but you could use something more "standard"

like using envsubst instead of needing to generate sed cmd for each var in vars.rst.

that means
1. have var.env file created from release.yaml
2. generate vars.rsh from 1
3. generate examples using vars from 1

example: in templates you will put something like: `repository: $NVIDIA_IPAM_REPO`

Reply:
Issue created:
https://github.com/Mellanox/network-operator-docs/issues/238